src/utils/webrtc/webrtc.js

```diff
@@ -417,7 +417,7 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 			callParticipantModel.set('speaking', (event.flags & PARTICIPANT.SIP_FLAG.SPEAKING) > 0)
 			callParticipantModel.set('audioAvailable', (event.flags & PARTICIPANT.SIP_FLAG.MUTE_MICROPHONE) === 0)
 			callParticipantModel.set('raisedHand', {
-				state: (event.flags & PARTICIPANT.SIP_FLAG.RAISE_HAND) === 0,
+				state: (event.flags & PARTICIPANT.SIP_FLAG.RAISE_HAND) !== 0,
```

Comment:
nice catch! I couldn't test back then so it didn't catch the eye

Reply:
yeah I saw it flashing by everytime the user muted and unmuted, that's how I noticed